src/prefect/cli/run.py

```diff
@@ -107,7 +115,11 @@ def cloud(name, project, version, watch, logs):
         click.secho("{} not found".format(name), fg="red")
         return
 
-    flow_run_id = client.create_flow_run(flow_id=flow_id)
+    loaded_params = None
+    if parameters:
+        loaded_params = json.load(open(parameters))
```

Comment:
this looks like it could leave the file open, we should probably use a context manager

Reply:
Ah yes this should be using a context manager 👍